Probably a CMake bug
${KDE_INSTALL_INCLUDEDIR_KF5}                         =  include/KF5
${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_INCLUDEDIR_KF5} = /mingw64/include/KF5

Before patch INTERFACE_INCLUDE_DIRECTORIES: "${_IMPORT_PREFIX}/include/KF5/KWindowSystem;/mingw64/include/KF5;${_IMPORT_PREFIX}/include/KF5"
When importing KF5::WindowSystem there is an error about non-existent path "/mingw64/include/KF5"
However ${_IMPORT_PREFIX}/include/KF5 is fines
This patch prevents "/mingw64/include/KF5" from being added

diff -Naur kwindowsystem-5.23.0.orig/src/CMakeLists.txt kwindowsystem-5.23.0/src/CMakeLists.txt
--- kwindowsystem-5.23.0.orig/src/CMakeLists.txt	2016-06-06 13:50:20.000000000 +0200
+++ kwindowsystem-5.23.0/src/CMakeLists.txt	2016-07-04 01:02:18.816288200 +0200
@@ -85,12 +85,6 @@
     target_include_directories(KF5WindowSystem PUBLIC ${XCB_XCB_INCLUDE_DIR} ${X11_X11_INCLUDE_PATH})
 endif()
 
-if(IS_ABSOLUTE "${KDE_INSTALL_INCLUDEDIR_KF5}")
-  target_include_directories(KF5WindowSystem INTERFACE "$<INSTALL_INTERFACE:${KDE_INSTALL_INCLUDEDIR_KF5}>" )
-else()
-  target_include_directories(KF5WindowSystem INTERFACE "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_INCLUDEDIR_KF5}>" )
-endif()
-
 set_target_properties(KF5WindowSystem PROPERTIES VERSION   ${KWINDOWSYSTEM_VERSION_STRING}
                                                  SOVERSION ${KWINDOWSYSTEM_SOVERSION}
                                                  EXPORT_NAME WindowSystem
